### PR TITLE
Fix GOBIN unset parameter

### PR DESCRIPTION
**Pull request by crazy-max**
* Fix GOBIN unset parameter (ref. #6)
* Add a simple [GitHub Action test workflow](https://github.com/crazy-max/xx/runs/744505240?check_suite_focus=true) to avoid regressions

image

### CODE REVIEW

File: golang/Dockerfile.test [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM --platform=${BUILDPLATFORM:-linux/amd64} golang:alpine AS builder
```
Comment (tonistiigi) [inline golang/Dockerfile.test:1]: The defaults seem unnecessary in here and in RUN.

File: golang/Dockerfile.test [hunk 2]
```diff
@@ -0,0 +1,8 @@
+FROM --platform=${BUILDPLATFORM:-linux/amd64} golang:alpine AS builder
+
+ARG TARGETPLATFORM
+ARG BUILDPLATFORM
+RUN printf "I am running on ${BUILDPLATFORM:-linux/amd64}, building for ${TARGETPLATFORM:-linux/amd64}\n$(uname -a)\n"
+
+COPY wrapper.sh /go/bin/go
+RUN go env
```
Comment (tonistiigi) [inline golang/Dockerfile.test:8]: Instead of separate file can you add it as a test stage

File: golang/Dockerfile.test (hunk 1 above)
Comment (crazy-max) [reply golang/Dockerfile.test]: Should be ok now

File: golang/Dockerfile.test (hunk 2 above)
Comment (crazy-max) [reply golang/Dockerfile.test]: Yeah, you're right. I merged it in the Dockerfile.

**Comment by tonistiigi** [comment]
`docker.io/tonistiigi/xx:golang@sha256:f806b9646c7dd81a50f9ec829cd896aa02d9aeb7782a80bc652dc9acdf5156fb` pushed

**Comment by crazy-max** [comment]
Thanks!

**Pull request merged**